tests/AssertTraitTest.php

```diff
@@ -13,41 +13,31 @@
 
 class AssertTraitTest extends \PHPUnit_Framework_TestCase
 {
-    private static function getSchema($filename)
-    {
-        return implode(DIRECTORY_SEPARATOR, [__DIR__, 'schemas', $filename]);
-    }
-
-    private static function getJson($filename)
-    {
-        return implode(DIRECTORY_SEPARATOR, [__DIR__, 'json', $filename]);
-    }
-
-    public function testAssertJsonSchema()
+    public function testAssertJsonMatchesSchema()
     {
         $content = json_decode('{"foo":123}');
 
-        AssertTraitImpl::assertJsonMatchesSchema(self::getSchema('test.schema.json'), $content);
+        AssertTraitImpl::assertJsonMatchesSchema(Utils::getSchemaPath('test.schema.json'), $content);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'EnricoStahn\JsonAssert\Tests\AssertTraitImpl' in method 'testAssertJsonMatchesSchema'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2256991340/issues/source?bid=3181605&fileBranchId=3182397#l20)